### ISSUE TITLE
replicate results on CUHK Avenue dataset

### ISSUE THREAD

**Issue by lyn1874** [cvlab-yonsei/MNAD#5]
Hi, Thanks a lot for your great work!

I want to replicate your results on the CUHK Avenue dataset, but I can only get around 70% anomaly detection accuracy. Then I give another read on the paper. I found there are several places that I am not quite sure:

1. I trained the model using PyTorch 1.1.0, with 60 epochs, initial learning rate 2e-4 which is decayed by the cosine annealing method. However, it seems that the Separateness jumps up and down, so I am wondering is it because of my learning rate too high?

2. On page 6, it's said that "We set initial learning rates to 2e-5 and 2e-4, respectively, for reconstruction and prediction tasks", but from what I have seen from the code, it seems that there is only one learning rate. 

Could you please give me some help with this? Thanks a lot in advance!

**Comment by hyunjp** [comment]
Hi, thanks for your attention to our work.

1. We set the initial learning rate to 2e-4, and used the cosine annealing method, too. We downloaded our uploaded public code again, and implemented it with the avenue dataset. We then got around 86~88% AUC.

Please check your dependencies or your dataset setting.

2. We only uploaded the code based on prediction method as we mentioned in **README.md**. 

Thank you.

**Comment by lyn1874** [comment]
Hi, thank you very much for your reply!

It turns out my own gt file is in a different format as the gt file that you uploaded, so my variable `label_list` is totally wrong. After I changed it, I got 85.69 accuracy which is reasonable I guess since I only run it once. 

Again, thank you very much!

**Comment by happycoding1996** [comment]
@hyunjp Thanks for your explanation. 

Is the variance acceptable 86~88?  

I have got a result of 86.7. How to get the result 88.5 reported in the paper?

**Comment by NeeluMadanCreate** [comment]
> Hi, thank you very much for your reply!
> 
> It turns out my own gt file is in a different format as the gt file that you uploaded, so my variable `label_list` is totally wrong. After I changed it, I got 85.69 accuracy which is reasonable I guess since I only run it once.
> 
> Again, thank you very much!

Hi lyn,

With the label file from the current repo, I am still getting an accuracy of 83.8. Could you please help to share your experience on that?

Thanks and Regards,
Neelu Madan
